CONTRIBUTING.adoc

```diff
@@ -66,9 +66,9 @@ export OS_OUTPUT_GOPATH=1
 
         $ mkdir -p $GOPATH/src/github.com/openshift
         $ cd $GOPATH/src/github.com/openshift
-        $ git clone git://github.com/<forkid>/origin  # Replace <forkid> with the your github id
+        $ git clone https://github.com/<forkid>/origin.git  # Replace <forkid> with the your github id
```

Comment:
most contributors use sshkey auth to be able to push to their github forks. https won't work for that.